lib/internal/fs.js

```diff
@@ -18,7 +19,7 @@ const {
 
 function assertEncoding(encoding) {
   if (encoding && !Buffer.isEncoding(encoding)) {
-    throw new Error(`Unknown encoding: ${encoding}`);
+    throw new errors.TypeError('ERR_INVALID_OPT_VALUE', 'encoding', encoding);
```

Comment:
I believe this and `assertEncoding` are superseded by https://github.com/nodejs/node/pull/11317
(Where I made the opposite argument 😉 https://github.com/nodejs/node/pull/11317#discussion_r123251060 )